content/coordination/mentoring/index.en.md

```diff
@@ -16,53 +16,51 @@ For mentors, this is an opportunity to help to build community, learn about the
 
 ### Who can be a mentor?
```

Comment:
should we move the formatting changes to another PR/commit? fine to drop this suggestion if it is not straightforward